src/components/header/SkinTonePicker.tsx

```diff
@@ -55,16 +59,52 @@ export function SkinTonePicker({
 
   const vertical = direction === SkinTonePickerDirection.VERTICAL;
 
+  const getHorizontalTranslation = ({
```

Comment:
In general, since this function is not crucial for understanding the flow of the component, I prefer putting it *after* the return statement - the magazine code organization approach.

If you change it to a named function, you can put it at the bottom of the component, but it will still be hoisted up.